### ISSUE TITLE
Handle duplicate <file>s in psalm-baseline

### ISSUE THREAD

**Issue by AndrolGenhald** [vimeo/psalm#7465]
I ran into an issue where, due to a bad merge, I had duplicated `<file>`s in my psalm baseline. Currently it looks like psalm just uses the second element and ignores the first. I think we should either allow this and merge them, or disallow it and throw an exception when parsing it.

**Comment by orklah** [comment]
Yeah. kinda related to https://github.com/vimeo/psalm/issues/6888. It's the same issue but for issueHandlers in config
